docs/MockFunctions.md

```diff
@@ -7,6 +7,12 @@ Mock functions allow you to test the links between code by erasing the actual im
 
 There are two ways to mock functions: Either by creating a mock function to use in test code, or writing a [`manual mock`](ManualMocks.md) to override a module dependency.
 
+:::tip
+
+When using `babel-jest`, calls to `disableAutomock()`, `enableAutomock()`, `unmock()` and `mock()` will automatically be hoisted to the top of the code block.
```

Comment:
Why do you think it should be mentioned in this page? Below on `jest.mock()` is used. Other API are not mentioned.

Mock function is the one returned by `jest.fn()`. Also after calling `jest.mock()` an automatic mock is created using the mock functions. Not sure I understood your idea, because hoisting does not have any direct affected on mock functions.

Reply:
it would be better to move it to the [Mocking Partials](https://jestjs.io/docs/mock-functions#mocking-partials) paragraph. There's a `jest.mock` call that mocks an import, and I believe it's hoisted there

Reply:
What about [Mock Modules](https://jestjs.io/docs/jest-object#mock-modules)? This chapter is all about the methods you trying to talk about. Or I missed something?